src/build.js

```diff
@@ -1,7 +1,53 @@
-var relativeImports = /import\s*{[a-zA-Z\,\s]+}\s*from\s*'\.\/[a-zA-Z\-]+';\s*/g;
```

Comment:
I modified this regex to group the import source so it could be reused, and also changed it to match any path starting with a `.`, to accommodate `../`, etc.